smart-contracts/.openzeppelin/kovan.json

```diff
@@ -1,175 +1,113 @@
 {
-  "contracts": {
-    "Unlock": {
+  "manifestVersion": "3.2",
+  "impls": {
+    "d862affc38f30c05541fe6bb9dcf43459bdcd003c1b971ad3df9a8825498c2c7": {
```

Comment:
@clemsos Tu me confirmes que les changements dans les `.openzeppelin` sont expected?

Reply:
I reverted all changes for files in `.zeppelin`. We can re-enact them using ZOS migration tool - for another PR